src/applications/gi-sandbox/components/profile/EstimatedBenefits.jsx

```diff
@@ -142,71 +164,93 @@ export default function EstimatedBenefits({ profile, outputs, calculator }) {
-            visible={outputs.totalPaidToYou.visible}
-            bold
-          />
-        </ul>
+    <>
+      <div className="medium-6 columns your-estimated-benefits">
+        <h3 id="estimated-benefits" tabIndex="-1">
+          Your estimated benefits
+        </h3>
+        <div aria-atomic="true" aria-live="polite" role="status">
+          {/* eslint-disable-next-line jsx-a11y/no-redundant-roles */}
+          <ul
+            className="out-of-pocket-tuition"
+            aria-label="Out-of-pocket tuition"
+            role="list"
+          >
+            <CalculatorResultRow
+              label="GI Bill pays to school"
+              value={outputs.giBillPaysToSchool.value}
+              visible={outputs.giBillPaysToSchool.visible}
+              screenReaderSpan={year}
+              bold
+            />
+            <CalculatorResultRow
+              label="Tuition and fees charged"
+              value={outputs.tuitionAndFeesCharged.value}
+              visible={outputs.tuitionAndFeesCharged.visible}
+            />
+            <CalculatorResultRow
+              label="Your scholarships"
+              value={outputs.yourScholarships.value}
+              visible={outputs.yourScholarships.visible}
+            />
+            <CalculatorResultRow
+              label="Out of pocket tuition"
+              value={outputs.outOfPocketTuition.value}
+              bold
+              visible={outputs.outOfPocketTuition.visible}
+            />
+          </ul>
+          {/* eslint-disable-next-line jsx-a11y/no-redundant-roles */}
+          <ul
+            className="total-paid-to-you"
+            aria-label="Total paid to you"
+            role="list"
+          >
+            <CalculatorResultRow
+              label="Housing allowance"
+              value={outputs.housingAllowance.value}
+              visible={outputs.housingAllowance.visible}
+              screenReaderSpan={month}
+              bold
+            />
+            <CalculatorResultRow
+              label={
+                <LearnMoreLabel
+                  text="Book stipend"
+                  onClick={() => dispatchShowModal('bookStipendInfo')}
+                  ariaLabel="Learn more about the book stipend"
+                />
+              }
+              id={'book-stipend'}
+              value={outputs.bookStipend.value}
+              visible={outputs.bookStipend.visible}
+              screenReaderSpan={
+                profile.attributes.type === 'ojt' ? month : year
+              }
+              bold
+            />
+            <CalculatorResultRow
+              label="Total paid to you"
+              value={outputs.totalPaidToYou.value}
+              visible={outputs.totalPaidToYou.visible}
+              bold
+            />
+          </ul>
+        </div>
+        <hr aria-hidden="true" />
+        {perTermSections()}
       </div>
-      <hr aria-hidden="true" />
-      {perTermSections()}
-    </div>
+      {isOJT && (
```

Comment:
This note does not visually match prototype

Reply:
If this needs to exist outside the background then it should be within a `<div className="medium-6 columns ">` along with current div which would then become just `<div className="your-estimated-benefits">`